Floofbot/Migrations/20200506001332_InitialCreate.cs

```diff
@@ -0,0 +1,164 @@
+                {
+                    ServerId = table.Column<ulong>(nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    IsOn = table.Column<bool>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_FilterConfigs", x => x.ServerId);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "FilteredWords",
+                columns: table => new
+                {
+                    Id = table.Column<int>(nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    ServerId = table.Column<ulong>(nullable: false),
+                    Word = table.Column<string>(nullable: true)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_FilteredWords", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "LogConfigs",
+                columns: table => new
+                {
+                    ServerId = table.Column<ulong>(nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    MessageUpdatedChannel = table.Column<ulong>(nullable: false),
+                    MessageDeletedChannel = table.Column<ulong>(nullable: false),
+                    UserBannedChannel = table.Column<ulong>(nullable: false),
+                    UserUnbannedChannel = table.Column<ulong>(nullable: false),
+                    UserJoinedChannel = table.Column<ulong>(nullable: false),
+                    UserLeftChannel = table.Column<ulong>(nullable: false),
+                    MemberUpdatesChannel = table.Column<ulong>(nullable: false),
+                    UserKickedChannel = table.Column<ulong>(nullable: false),
+                    UserMutedChannel = table.Column<ulong>(nullable: false),
+                    UserUnmutedChannel = table.Column<ulong>(nullable: false),
+                    IsOn = table.Column<bool>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_LogConfigs", x => x.ServerId);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "NicknameAlertConfigs",
+                columns: table => new
+                {
+                    ServerId = table.Column<ulong>(nullable: false)
+                        .Annotation("Sqlite:Autoincrement", true),
+                    Channel = table.Column<ulong>(nullable: false),
+                    IsOn = table.Column<bool>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_NicknameAlertConfigs", x => x.ServerId);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Tags",
```

Comment:
lol this is nice.
what I could do in the future is add some code to slowly migrate the old tag format to the new tag format, since 
`${old tagname} = ${new tagname}:${serverId}` anyway.

Reply:
I was thinking about doing some sublime magic on an exported CSV